### PR TITLE
Develop rain 1724

### PR DESCRIPTION
**Pull request by praneeth-egov**

### CODE REVIEW

**Review by kavi-egov — CHANGES_REQUESTED** [review]
The old tenantid should be there

File: property-services/src/main/java/org/egov/pt/repository/PropertyRepository.java [hunk 1]
```diff
@@ -63,28 +64,33 @@
...
-			builder.append(" where tenantid=?");
-			preparedStmtList.add(criteria.getTenantId());
+		if (!ObjectUtils.isEmpty(criteria.getTenantIds()) && !ObjectUtils.isEmpty(criteria.getFromDate()) && !ObjectUtils.isEmpty(criteria.getToDate())) {
+			builder.append(" where tenantid IN (:tenantIds) BETWEEN fromDate=? AND toDate=? ");
+			preparedStmtList.add(criteria.getTenantIds());
+			preparedStmtList.add(criteria.getFromDate());
+			preparedStmtList.add(criteria.getToDate());
 		}
-		String orderbyClause = " order by lastmodifiedtime,id offset ? limit ?";
+		String orderbyClause = " order by lastmodifiedtime,id offset ? limit ? ";
```
Comment (kavi-egov) [inline property-services/src/main/java/org/egov/pt/repository/PropertyRepository.java:93]: don't use id.  use propertyid.

**Review by kavi-egov — COMMENTED** [review]
why are there two methods in propertyRepository. fetch ids and get property id.

File: property-services/src/main/java/org/egov/pt/repository/PropertyRepository.java (hunk 1 above)
Comment (praneeth-egov) [reply property-services/src/main/java/org/egov/pt/repository/PropertyRepository.java:93]: methods fetchIds and getPropertyId were there from the start. I haven't added those methods

**Comment by shashwat-egov** [comment]
@praneeth-egov Why are there 4 commits? Make sure there is only one commit in the PR which contains all the changes that you've made.

**Pull request closed without merging**